datasets/paws-x/README.md

```diff
@@ -0,0 +1,206 @@
+---
+annotations_creators:
+- expert-generated
+- machine-generated
+language_creators:
+- expert-generated
+- machine-generated
+languages:
+- de
+- en
+- es
+- fr
+- ja
+- ko
+- zh
+licenses:
+- other-The dataset may be freely used for any purpose, although acknowledgement of
```

Comment:
This should only be `other`, the free text description needs to be in the paragraph at the end of the file

Reply:
Hi @yjernite,
I made that change in README and when I pushed the code few tests failed. Can you please look into it? Thanks!